packages/create-bison-app/template/.github/workflows/main.js.yml.ejs

```diff
@@ -100,71 +100,3 @@ jobs:
         with:
           name: cypress-videos
           path: cypress/videos
-
-  ###############
```

Comment:
Just want to double check that we really want to remove the Vercel deployment here too? I think that might be a good feature to keep in Bison if its working well. Setting up the CI can be time consuming so I think its really convenient when its provided out of the box.

If we do want to remove it, I think we'll have to remove the following lines from the main readme and also update the [CI](https://github.com/echobind/bisonapp/blob/GH179-Heroku-Setup-Docs/docs/ci.md) docs:



![Screen Shot 2021-07-21 at 11 42 12 AM](https://user-images.githubusercontent.com/1087679/126519352-e099bffa-029b-4039-aa1d-adbd0dabe0df.png)

Reply:
@kgajera @cball -- From my last converstion, Chris mentioned taking it all out. 
To your point, if we are going to partner -- maybe that's the one flow we DO support. 

Probalby easier to find some five minutes for the three of us to chat. 
There is a lot of good code here.